### DISCUSSION TITLE
Logo and Login-Background-Image not visible to Guests

### DISCUSSION THREAD

**Discussion by germeshausen** [requarks/wiki#2759] [Help / Questions] [answered]
**Describe the bug**
We have changed the permissions for the Guest-Group to limit the access to the home. If so we mentioned that the logo is not loaded and also the login-background image cannot be loaded.

**To Reproduce**
Define a pagerule which differs from 'path starts with' '/'
or the path for a exact match
or allowing Guests to read only pages with a specific tag.

**Expected behavior**
logo and background image should not be affected by page rules as they belong to the website and do not to the content.

**Host Info (please complete the following information):**
 - OS: Windows Server 2019
 - Wiki.js version: 2.5.170
 - Database engine: postgres 12.4
- node.js 14.8.0

**Comment by lewer** [comment, marked as answer]
On Admin > Security, where you configure the background login image, it's said:

> Note that the Guests group must have read-access to the selected image!

so you should probably create a custom directory in assets where you put your images and give Guests group read access to it.
